### PR TITLE
Add patch with support for Boost 1.63

### PR DESCRIPTION
**Pull request by tadeu**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,8 +18,12 @@ source:
   fn: cmake-{{ version }}-win64-x64.zip                                                   # [win64]
   sha256: 659ecb8207e1266786188c7eaf45308458ba5f719c985970f6f55ec0b5a96746                # [win64]
 
+  patches:
+    - FindBoost-Add-support-for-1.63.patch  # [not win]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Looks good to me. Maybe just add a note that the patch is already upstream and maybe the link to the diff:

https://gitlab.kitware.com/cmake/cmake/merge_requests/361.patch

Comment (tadeu) [inline recipe/meta.yaml]: Ok, I've added the link inside the patch header, but as a comment in the `yaml` seems more visible!

**Comment by ocefpaf** [comment]
We already know this is passing :wink:

**Comment by tadeu** [comment]
Thanks 👍

**Pull request merged**